.github/workflows/build.yaml

```diff
@@ -0,0 +1,71 @@
+name: Main build
+
+on:
+    push:
+        branches: [main]
+
+env:
+  REGISTRY: ghcr.io
+  NAMESPACE: galasa-dev
+  IMAGE_TAG: main
+    
+jobs:
+    build-extensions:
+        name: build and Test Extensions 
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout Code
+              uses: actions/checkout@v4
+            
+            - name: Setup JDK 
+              uses: actions/setup-java@v4
+              with:
+                java-version: '11'
+                distribution: 'semeru'
+            
+            - name: Print githash
+              run: |
+                echo $GITHUB_SHA > ./extensions.githash
+            
+            - name: Setup Gradle
+              uses: gradle/actions/setup-gradle@v3
+              with:
+                gradle-version: 6.9.2
+  
+            - name: Test and Build Extensions using gradle
+              env:
+                GITHUB_ACTOR: ${{github.actor}}
+                GITHUB_TOKEN: ${{ github.token }}

```

Comment:
I think we are missing these:
```
          ORG_GRADLE_PROJECT_signingKeyId:  ${{ secrets.GPG_KEYID }}
          ORG_GRADLE_PROJECT_signingKey: ${{ secrets.GPG_KEY }}
          ORG_GRADLE_PROJECT_signingPassword: ${{ secrets.GPG_PASSPHRASE }}
```

Reply:
I think the artefact signing was absent from the tekton builds , therefore I removed these.

Reply:
Okay , so signing is present in java.gradle file, I'll make the changes.

Reply:
done